src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationCheckSwitchExpressionDeclaration.java

```diff
@@ -0,0 +1,33 @@
+package com.puppycrawl.tools.checkstyle.checks.indentation.indentation;             //indent:0 exp:0
+                                                                                  //indent:82 exp:82
+/* Config:                                                                          //indent:0 exp:0
+ *                                                                                  //indent:1 exp:1
+ * basicOffset = 4                                                                  //indent:1 exp:1
+ * braceAdjustment = 0                                                              //indent:1 exp:1
+ * caseIndent = 4                                                                   //indent:1 exp:1
+ * throwsIndent = 4                                                                 //indent:1 exp:1
+ * arrayInitIndent = 4                                                              //indent:1 exp:1
+ * lineWrappingIndentation = 8                                                      //indent:1 exp:1
+ * forceStrictCondition = false                                                     //indent:1 exp:1
+ */                                                                                 //indent:1 exp:1
+public class InputIndentationCheckSwitchExpressionDeclaration {                     //indent:0 exp:0
+                                                                                  //indent:82 exp:82
+    public static void main(String... args) {                                       //indent:4 exp:4
+        if (args.length < 2) {                                                      //indent:8 exp:8
+            return;                                                               //indent:12 exp:12
+        }                                                                           //indent:8 exp:8
+        String requestId = switch (args[0]) {                                       //indent:8 exp:8
+            case "CREATE" -> "RANDOM";                                            //indent:12 exp:12
+                case "UPDATE" -> args[1];                                    //indent:16 exp:12 warn
+                default -> "DEFAULT";                                        //indent:16 exp:12 warn
+        };                                                                          //indent:8 exp:8
+        requestId = switch (args[0]) {                                              //indent:8 exp:8
+            case "CREATE" -> "RANDOM";                                            //indent:12 exp:12
+                case "UPDATE" -> args[1];                                    //indent:16 exp:12 warn
+                default -> "DEFAULT";                                        //indent:16 exp:12 warn
+        };                                                                          //indent:8 exp:8
+        System.out.println(requestId);                                              //indent:8 exp:8
```

Comment:
few things.
1. lines 25-28 are duplicated, please remove them since they do not show any new violations
2. Sample code is not required to be correct, so there is no need for input args check, println and so on.
3. There is file InputIndentationCheckSwitchExpressionCorrect, why you cannot add your example there? Is there config difference?

Reply:
> lines 25-28 are duplicated, please remove them since they do not show any new violations

I asked contributor to add workaround from https://github.com/checkstyle/checkstyle/issues/9302#issuecomment-789431828 to make sure that violations were consistent; I should have shared a link, apologies.